src/vcSceneLayer.cpp

```diff
@@ -506,7 +514,7 @@ udResult vcSceneLayer_LoadGeometryData(const vcSceneLayer *pSceneLayer, vcSceneL
             const float lonRadians = (float)UD_DEG2RAD(pNode->latLong.x + posLatLong.x);
             const udFloat4x4 ECEFToNED = udFloat4x4::create(
               -udSin(latRadians) * udCos(lonRadians), -udSin(lonRadians), -udCos(latRadians) * udCos(lonRadians), 0,
-              -udSin(latRadians) * udSin(lonRadians),  udCos(lonRadians), -udCos(latRadians) * udSin(lonRadians), 0,
+              -udSin(latRadians) * udSin(lonRadians), udCos(lonRadians), -udCos(latRadians) * udSin(lonRadians), 0,
```

Comment:
Did you intend to remove this alignment?

Reply:
Unsure to be honest.